src/h2load.cc

```diff
@@ -187,6 +224,17 @@ Client::Client(Worker *worker, size_t req_todo)
 
   wev.data = this;
   rev.data = this;
+
+  if (worker->config->conn_inactivity_timeout > 0) {
+    conn_inactivity_watcher.data = this;
+    ev_init(&conn_inactivity_watcher, conn_inactivity_timeout_cb);
+    conn_inactivity_watcher.repeat = worker->config->conn_inactivity_timeout;
+  }
+  if (worker->config->conn_active_timeout > 0) {
+    conn_active_watcher.data = this;
+    ev_timer_init(&conn_active_watcher, conn_active_timeout_cb,
+                  worker->config->conn_active_timeout, 0);
+  }
```

Comment:
It would be simpler to just initialize both timers regard less of configuration.
It is harmless unless they are actually started.